CHANGELOG.md

```diff
@@ -12,96 +12,103 @@ Bugfixes:
 
 Other improvements:
 
+## [v6.0.0](https://github.com/purescript-node/purescript-node-http/releases/tag/v6.0.0) - 2021-MONTH-DAY
+
+Breaking changes:
+  - Updated dependencies for PureScript 0.14 (#31)
+
+New features:
+  - Added `onUpgrade` to allow users to listen to and respond to HTTP upgrades (#33)
+
+Other improvements:
+  - Migrated CI to GitHub Actions, updated installation instructions to use Spago, and migrated from `jshint` to `eslint` (#30)
+  - Added a changelog and pull request template (#34)
+
 ## [v5.0.1](https://github.com/purescript-node/purescript-node-http/releases/tag/v5.0.1) - 2019-05-28
```

Comment:
Could you add a release section for changes made in [`v0.5.2`](https://github.com/purescript-node/purescript-node-http/compare/v5.0.1...v5.0.2)?